src/components/attendee/VideoHuddle/VideoHuddle.tsx

```diff
@@ -1,57 +1,35 @@
-import React, { useContext, useState } from "react";
+import React from "react";
 
-import { useVideoRoomState } from "hooks/twilio/useVideoRoomState";
-
-import { VideoParticipant } from "components/organisms/Video";
+import { VideoCommsParticipant } from "../VideoComms/VideoCommsParticipant";
 
 import { useVideoHuddle } from "./useVideoHuddle";
 
 import styles from "./VideoHuddle.module.scss";
 
-interface VideoHuddleContextType {
-  channelId?: string;
-  setChannelId: (channelId?: string) => void;
-}
-
-export const VideoHuddleContext = React.createContext<VideoHuddleContextType>({
-  setChannelId: () => {},
-});
-
 interface VideoHuddleProps {
   userId: string;
 }
 
 export const VideoHuddle: React.FC<VideoHuddleProps> = ({ userId }) => {
-  const { channelId } = useContext(VideoHuddleContext);
+  const { inHuddle } = useVideoHuddle();
```

Comment:
Shouldn't at least the `userId` be passed in here? Will there ever be a case for not the currently used user?